SECENG-6135 Add in 403 response for HTTP API docs about WAF (#358)
amplitude/amplitude-docs: content/collections/api/en/http-v2.md

| `throttled_devices`                | Object. A map from device_id to its current events per second rate, for all devices that exceed the app's current threshold. |
| `throttled_events`                 | [integer]. Array of indexes in the events array indicating events whose `user_id` or `device_id` got throttled               |

### 403 (forbidden)

> [403 Forbidden](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3). The request is blocked by our Web Application Firewall (WAF).

Reviewer: ```suggestion

[403 Forbidden](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3). The request is blocked by Amplitude's Web Application Firewall (WAF).
```

Possible reasons for the response:

- The request contains invalid values within the Header, Body, and/or URI that match our security filters

> - The request is from a sanctioned region that we are not allowed to accept requests from

Reviewer: ```suggestion
- The request is from a sanctioned region from which Amplitude isn't allowed to accept requests.
```
